evaltools/utils/JSON.py

```diff
@@ -1,20 +1,56 @@
 
 import json
-from typing import List, Union
+from pydantic import BaseModel, ValidationError, validator
+from typing import List, Union, Any
 
-class JSON(object):
+class JSON(BaseModel):
```

Comment:
Is `JSON` the best name for this class (since `json` is a built-in library that people can import)?